src/core/logical_layers/renderers/MultivariateRenderer/popup.ts

```
@@ -0,0 +1,12 @@
+import { createRoot } from 'react-dom/client';
+import { PopupMultivariate as PopupMultivariate } from './components/PopupMultivariate';
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)